src/App.js

```diff
@@ -10,13 +10,20 @@ class App extends Component {
     this.state = {
       test: false
     }
+    this.resetGame = this.resetGame.bind(this);
+  }
+
+  resetGame(e) {
+    e.preventDefault();
+    this.setState({ test: true })
   }
 
   render() {
     return (
       <div className="page-wrapper">
-        <Navbar />
-        <Board /> 
+        <div>{ this.state.test }</div>
+        <Navbar resetGame={ this.resetGame }/>
+        <Board shouldRefresh={ this.state.test } /> 
```

Comment:
I pass the new state (i.e. "I should refresh now, because I clicked the button") to the Board component.